### PR TITLE
Add page with language stats

### PR DESCRIPTION
**Pull request by nickygerritsen**
As discussed before.

Screenshots:
Screenshot 2024-08-11 at 15 21 40
Screenshot 2024-08-11 at 15 21 46

There is a button above the language graph to link to this page:
image

I made it two columns next to each other first, but expanding the teams will make a lot of teams two lines, so I didn't do that  in the end.

Feedback on what to show or how to show it is welcome.

### CODE REVIEW

**Comment by vmcj** [comment]
I would also be interested to see for which problems the language was used:
Lang:
1 team
Used for solving problems: [A][C]

**Comment by nickygerritsen** [comment]
Stealing some of @meisterT his awesome CSS skills to produce this:
image

**Comment by vmcj** [comment]
> Stealing some of @meisterT his awesome CSS skills to produce this: image

This will make that we need to fix the WCAG tests for jury (probably) as this has not enough contrast. As this is by design I'm fine with it.

**Comment by vmcj** [comment]
The only thing missing for Astana is an easy export of this so we can directly share this when we start with EOC with whoever needs this? Is the plan to just save this page or should we also expose this via the API?

**Comment by meisterT** [comment]
Keep it simple: Just print it and hand it over.

File: webapp/src/Service/StatisticsService.php [hunk 1]
```diff
@@ -514,6 +517,103 @@ public function getGroupedProblemsStats(
...
+                }
+                $languageStats[$language->getLangid()]['problems_attempted'][$s->getProblem()->getProbId()] = $s->getContestProblem();
+            }
+        }
+
+        foreach ($languageStats as &$languageStat) {
+            usort($languageStat['teams'], static fn(
+                Team $a,
+                Team $b
+            ) => ($a->getLabel() ?: $a->getExternalid()) <=> ($b->getLabel() ?: $b->getExternalid()));
```
Comment (meisterT) [inline webapp/src/Service/StatisticsService.php:601]: Sort by number of problems solved using this language instead?

Comment (nickygerritsen) [reply webapp/src/Service/StatisticsService.php:601]: Do you think that makes more sense than by team ID? Won't it look confusing if the teams are in arbitrary order?

Or do you propose to also show the number of solved problems per team per language? What if a team didn't solve but did try a problem in the language?

**Comment by nickygerritsen** [comment]
After talking to @meisterT I improved it to this:
image

(don't mind the double `in`, I fixed that)

**Review by meisterT — APPROVED** [review]

**Pull request merged**